src/test/migrations/index.ts

```diff
@@ -17,8 +17,27 @@ let migrationSteps: Array<string>
 let previousMigrations: Array<string> = []
 const executedMigrations: Array<string> = []
 
+const tableDDL = `
```

Comment:
TODO: For some reasons some already ran migrations are running. Need to resolve this to open from draft